docs/content/release_notes.rst

```diff
@@ -1,6 +1,20 @@
 Release Notes
 =============
 
+
+Release 0.6.3 - 2022/02/09
+--------------------------
+
+**(Hot fixes)**
+
+ - Fix limitations for unrecognized ui.json keys
+ - Fix Mag/grav inversion crash for:
+    - Selection of multi-components grav/mag data
+    - Constant topography value option
+    - Min/max values for inducing field angles
+    - Update ui.json for default bool index
```

Comment:
We also added Points/Curves to the apps last sprint (GEOPY-432).  Seems like it should be included here, but I merged into 0.7.0 branch for some reason.